.gitignore

```diff
@@ -280,6 +280,7 @@ sftp-config.json
 
 ## Directory-based project format:
 .idea/
+nbproject/
```

Comment:
for Netbeans please :)

Reply:
It's not needed, with git you can create a global gitignore file in your home directory
`~/.gitignore` we should only use what we have in this project :thinking:

Reply:
you are right, but as there was `nbproject/private` ignore already, I simplified it and both usecases should be handled